### DISCUSSION TITLE
Support PIPELINING extension?

### DISCUSSION THREAD

**Discussion by gene-hightower** [postalserver/postal#1874] [Feature suggestions]
Any hope for supporting RFC-2920 "SMTP Service Extension for Command Pipelining" -- it could improve network utilization and overall performance quite a bit.

**Comment by willpower232** [comment]
If someone wanted to make a PR for it then sure but Postals primary focus is transactional email where messages don't tend to be in significantly sized batches so I don't think this would make a big difference.
